cmdstanpy_tutorial.ipynb

```diff
@@ -324,23 +621,89 @@
     "    + high R-hat values\n",
     "   + See the Stan reference manual section on [posterior analysis](https://mc-stan.org/docs/reference-manual/analysis-chapter.html) for further details.\n",
     "\n"
-   ],
-   "metadata": {}
+   ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Processing csv files: /var/folders/db/4jnggnf549s42z50bd61jskm0000gq/T/tmpki9l1wdc/bernoulli-20210928140611-1-js2crwjc.csv, /var/folders/db/4jnggnf549s42z50bd61jskm0000gq/T/tmpki9l1wdc/bernoulli-20210928140611-2-bs65mqc0.csv, /var/folders/db/4jnggnf549s42z50bd61jskm0000gq/T/tmpki9l1wdc/bernoulli-20210928140611-3-ovs93_pk.csv, /var/folders/db/4jnggnf549s42z50bd61jskm0000gq/T/tmpki9l1wdc/bernoulli-20210928140611-4-b8uhke8q.csv\n",
+      "\n",
+      "Checking sampler transitions treedepth.\n",
+      "Treedepth satisfactory for all transitions.\n",
+      "\n",
+      "Checking sampler transitions for divergences.\n",
+      "No divergent transitions found.\n",
+      "\n",
+      "Checking E-BFMI - sampler transitions HMC potential energy.\n",
+      "E-BFMI satisfactory.\n",
+      "\n",
+      "Effective sample size satisfactory.\n",
+      "\n",
+      "Split R-hat values satisfactory all parameters.\n",
+      "\n",
+      "Processing complete, no problems detected.\n"
+     ]
+    }
+   ],
    "source": [
-    "fit.diagnose()"
+    "diags = fit.diagnose()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Processing csv files: /var/folders/db/4jnggnf549s42z50bd61jskm0000gq/T/tmpki9l1wdc/bernoulli-20210928140611-1-js2crwjc.csv, /var/folders/db/4jnggnf549s42z50bd61jskm0000gq/T/tmpki9l1wdc/bernoulli-20210928140611-2-bs65mqc0.csv, /var/folders/db/4jnggnf549s42z50bd61jskm0000gq/T/tmpki9l1wdc/bernoulli-20210928140611-3-ovs93_pk.csv, /var/folders/db/4jnggnf549s42z50bd61jskm0000gq/T/tmpki9l1wdc/bernoulli-20210928140611-4-b8uhke8q.csv\n",
+      "\n",
+      "Checking sampler transitions treedepth.\n",
+      "Treedepth satisfactory for all transitions.\n",
+      "\n",
+      "Checking sampler transitions for divergences.\n",
+      "No divergent transitions found.\n",
+      "\n",
+      "Checking E-BFMI - sampler transitions HMC potential energy.\n",
+      "E-BFMI satisfactory.\n",
+      "\n",
+      "Effective sample size satisfactory.\n",
+      "\n",
+      "Split R-hat values satisfactory all parameters.\n",
+      "\n",
+      "Processing complete, no problems detected.\n",
+      "\n"
+     ]
+    }
    ],
+   "source": [
+    "print(diags)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
    "outputs": [],
-   "metadata": {}
+   "source": []
```

Comment:
We should delete this trailing blank cell

Reply:
done